data/json/items/comestibles/carnivore.json

```diff
@@ -1898,5 +1898,58 @@
     "fun": 6,
     "weight": "298 g",
     "flags": [ "EATEN_HOT" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "meat_mi-go",
+    "category": "other",
+    "name": { "str": "chunk of mi-go meat", "str_pl": "chunks of mi-go meat" },
+    "weight": "238 g",
+    "color": "pink",
+    "spoils_in": "90 days",
+    "//": "It's so alien few micro organisms are able to degrade it.",
+    "use_action": [ "POISON" ],
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 0,
+    "description": "A piece of mi-go meat that's obviously alien, given its pink color, odd odor, and strange texture.  It would not be advisable to try to eat it.",
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "mi-go_flesh" ],
+    "volume": "250 ml",
+    "fun": -10,
+    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE" ]
+  },
+  {
+    "id": "blood_mi-go",
+    "type": "COMESTIBLE",
+    "copy-from": "meat_mi-go",
+    "name": { "str_sp": "mi-go blood" },
+    "description": "Mi-go blood that's obviously alien, given its pink color and strange odor.  It would not be advisable to try to eat it.",
+    "comestible_type": "DRINK",
+    "looks_like": "blood",
+    "symbol": "~",
+    "material": "mi-go_blood",
+    "charges": 1,
+    "phase": "liquid"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "bone_mi-go",
+    "category": "other",
+    "name": { "str": "mi-go bone" },
+    "weight": "350 g",
+    "color": "light_green",
+    "use_action": [ "POISON" ],
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 0,
+    "description": "A bone from a mi-go.  It's stronger than normal bone, so it might have some uses if you could figure one out, which you doubt you could, but it's unlikely it can replace any chemical usages for normal bone.",
```

Comment:
```suggestion
    "description": "A bone, or at least a kind of similar structure, from inside a mi-go.  It's strong and springy, somewhat like cartilage.  It's also a deep green color, and probably not at all similar to human bone.",
```

Reply:
The "probably not at all similar" part doesn't make sense to me.

What about "An unrecognized structural element from inside a mi-go. It's strong and springy, somewhat like cartilage, but a bit stiffer. It's also a deep green color, and might possibly serve as a scaffolding element similar to bone."

Reply:
The not similar to human bone bit was serving the same role as your chemistry comment, just clarifying that it should not be used in other bone recipes.

"might possibly serve as a scaffolding element similar to bone" --> "might serve a scaffolding role, like bone" and you're good I think